pr-data.csv

```diff
@@ -1391,14 +1392,15 @@ https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-s
 https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-connectors/flink-connector-kafka-base,org.apache.flink.table.descriptors.KafkaTest.testValidation,ID,,,https://github.com/TestingResearchIllinois/idoft/issues/8
 https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-table/flink-table-common,org.apache.flink.table.expressions.ExpressionTest.testValueLiteralString,ID,Accepted,https://github.com/apache/flink/pull/11269,
-https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-table/flink-table-planner,org.apache.flink.table.expressions.MapTypeTest.testMapField,ID,,,
+https://github.com/apache/flink,1845aa8e6fc094ed69a145b50ac295e058d8c695,flink-table/flink-table-planner,org.apache.flink.table.expressions.MapTypeTest.testMapField,ID,,,
+https://github.com/apache/flink,1845aa8e6fc094ed69a145b50ac295e058d8c695,flink-table/flink-table-planner,org.apache.flink.table.planner.expressions.ScalarFunctionsTest.testPi,ID,,,
 https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-table/flink-table-planner,org.apache.flink.table.expressions.MapTypeTest.testMapLiteral,ID,,,
 https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-table/flink-table-planner,org.apache.flink.table.expressions.MapTypeTest.testMapTypeCasting,ID,,,
-https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-table/flink-table-planner,org.apache.flink.table.expressions.SqlExpressionTest.testValueConstructorFunctions,ID,,,
+https://github.com/apache/flink,1845aa8e6fc094ed69a145b50ac295e058d8c695,flink-table/flink-table-planner,org.apache.flink.table.expressions.SqlExpressionTest.testValueConstructorFunctions,ID,,,
 https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-connectors/flink-connector-hive,org.apache.flink.table.functions.hive.HiveGenericUDFTest.testMap,ID,Accepted,https://github.com/apache/flink/pull/11052,
-https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-table/flink-table-planner-blink,org.apache.flink.table.planner.expressions.MapTypeTest.testMapField,ID,,,
-https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-table/flink-table-planner-blink,org.apache.flink.table.planner.expressions.MapTypeTest.testMapTypeCasting,ID,,,
-https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-table/flink-table-planner-blink,org.apache.flink.table.planner.expressions.ScalarFunctionsTest.testPi,ID,,,
+https://github.com/apache/flink,23c9b5ac50d04d28a34a87c78eb2d3331c06b74b,flink-table/flink-table-planner-blink,org.apache.flink.table.planner.expressions.MapTypeTest.testMapField,ID,MovedOrRenamed,https://github.com/apache/flink/pull/16386,https://github.com/apache/flink/pull/16386/commits/1845aa8e6fc094ed69a145b50ac295e058d8c695
```

Comment:
Remove the PR link from `MovedOrRenamed` and leave just the note for the commit that moved/renamed the test.

Reply:
In the commit that you listed I don't see the test being moved/renamed just some `pom.xml` files changed. What commit actually moved the test?

Reply:
The module was renamed! Tests were not moved!

Reply:
If the module *name* changed in `pom.xml` but the module *directory* wasn''t renamed, then the test remained in the same place. Maybe another commit moved the test class file from one directory to another. Or maybe I'm missing something. When was the test class itself moved?

Reply:
The test class hasn't changed "org.apache.flink.table.planner.expressions.MapTypeTest.testMapField", Only the module "flink-table/flink-table-planner-blink" has been renamed to "flink-table/flink-table-planner".

Reply:
I don't even see commit `1845aa8e6fc094ed69a145b50ac295e058d8c695` in `apache/flink`:
```
$ git show --name-status 1845aa8e6fc094ed69a145b50ac295e058d8c695
fatal: bad object 1845aa8e6fc094ed69a145b50ac295e058d8c695
```

Reply:
My bad, this was the commit ab70dcf, you can try it again on this. Ill update the pr-data if this is good enough

Reply:
Fine. I see now `R100    flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/expressions/MapTypeTest.scala       flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/expressions/MapTypeTest.scala`

Please update this in the PR and maybe split into separate PRs so we can separately review these `MovedOrRenamed` vs. the other changes.

Reply:
done! ill update the pr-data csv with the new tests in another PR

Reply:
Do you want to keep in this PR only these changes that we already discussed and move other changes in another PR?

Reply:
yes!

Reply:
Please feel free to revise this PR (just don't close it and reopen only new ones -- you can move from here whatever you want to another PR).

Reply:
these are all the changes for required for these tests, you can close this if it looks good to you